docs/maui/native-library-interop/index.md

```diff
@@ -0,0 +1,90 @@
+
+A key benefit of Native Library Interop is its effectiveness with simple API surfaces. When wrappers involve only primitive types that .NET supports, existing binding tools can generate reliable definitions with minimal manual intervention, which is often required for traditional bindings. This makes the process straightforward, especially since wrapper API implementation typically follows SDK documentation, and often allows direct copying from vendor documentation.
+
+While the initial setup may be more intricate, managing updates to underlying SDKs generally requires less effort. Updates often involve simply adjusting the version and rebuilding the project. Even if breaking changes occur in the API surfaces or SDKs, the wrapper API surface and .NET application's usage are more likely to remain stable, requiring fewer adjustments compared to traditional bindings.
+
+In summary, Native Library Interop provides several benefits:
+
+- Simplifies following SDK documentation with native languages and tools
+- Requires less manual intervention to create working bindings
+- Makes maintenance easier and reduces the frequency of necessary updates
+- Enhances isolation of the app from changes in underlying SDKs
+
+Although resolving dependency chains (particularly on Android) may require similar effort as traditional bindings, the streamlined implementation and maintenance advantages make Native Library Interop an appealing choice for many projects.
+
+### Understanding Maui.NativeLibraryInterop
+
+A notable challenge with creating and maintaining bindings created via Native Library Interop is manually coalescing the native projects, their native dependencies, build outputs, and the .NET Binding library project. **Maui.NativeLibraryInterop** helps you jumpstart the process by building from and customizing the samples for your own app's needs.
+
+Part of this includes orchestrating parts of the build process through MSBuild invocations. This can include:
+
+- Resolving or downloading native SDK dependencies
+- Building the native slim binding project and its dependencies
+- Moving the requisite native artifacts to the expected working directory
+- Generating the API definition for the binding library project
+
+The binding build process is extended to obtain and build native SDK dependencies by adding the `CommunityToolkit.Maui.NativeLibraryInterop.BuildTasks` NuGet package to your binding project:
+
+```xml
+<ItemGroup>
+    <PackageReference Include="CommunityToolkit.Maui.BindingExtensions" Version="0.0.1-pre1" />
```

Comment:
We'll want to update this to the new package name: 
```suggestion
    <PackageReference Include="CommunityToolkit.Maui.NativeLibraryInterop.BuildTasks" Version="0.0.1-pre1" />
```

Reply:
Not to throw a spanner in the works but wasn't there a big move to shift away from using the word _native_ and use _platform_ instead?

Reply:
@bijington thanks for bringing this up! We definitely need to be careful about when to use _native_ and this is something I personally have thought a lot about and had many conversations with folks and experts about. Ultimately, context matters - as I have been told to understand it, in this context, and today, it would be ok to use "native library" since it is always _with_ "library" and "native library" is an actual, industry-wide, accepted term.

of course, when the term is used in various different contexts and standing alone, it's much easier to just replace _all_ of it, which is why we replaced all instances of the term previously in the maui repo

Reply:
I'm with you @bijington, I believe **PlatformLibraryInterop** makes more sense, It’s a more modern and expressive way to call native functions. Also, it brings clarify and consistency. 

Just my grain of salt. 😃 

cc: @rachelkang @pjcollins